driver/src/bin/global_helpers.rs

```diff
@@ -0,0 +1,14 @@
+pub const ACCOUNTS: i32 = 8;
```

Comment:
nit: I would call this file `model.rs` or `defines.rs` something similar. Right now there are no helper functions in here, and global doesn't sound great either. 
Eventually we might want to read ACCOUNTS/TOKENS etc from environment variables so that we can ensure that all parts are running with the same configuration.